src/lib/detect/detect-breakpoint.ts

```diff
@@ -105,15 +105,15 @@ const updateBreakpoint = (breakpoint: SourceBreakpoint): void => {
  * using getViewPort, which utilizes window.innerWidth which causes a reflow.
  */
 const initMediaQueryListeners = () => {
-	Object.entries(sourceBreakpoints).forEach(([bp], index, bps) => {
```

Comment:
The order of the keys in this object have changed (which is not guaranteed anyway), `sourceBreakpointNames` makes more sense as it's an array.